apps/user-profile/src/components/modules/OnboardingLandingPage/OnboardingLandingPage.js

```diff
@@ -1,272 +1,385 @@
+          const reader = new FileReader();
+          reader.onloadend = () => {
+            setUploadedProfilePicture(reader.result);
+          };
+          reader.readAsDataURL(file);
+        } else {
+          alert('The file size must be less than 200MB.');
         }
+      } else {
+        alert('Please select a valid image file (PNG or JPEG).');
+      }
     }
+  }
 
-    function onFirstNameChange(e) {
-        console.log(person);
-        setPerson({ ...person, firstName: e.target.value, firstNameTouched: true });
-        console.log("VALIDATION: ", formValidation)
-    }
+  function onFirstNameChange(e) {
+    console.log(person);
+    setPerson({ ...person, firstName: e.target.value, firstNameTouched: true });
+    console.log('VALIDATION: ', formValidation);
+  }
 
-    function onLastNameChange(e) {
-        console.log(person);
-        setPerson({ ...person, lastName: e.target.value, lastNameTouched: true });
-        console.log("VALIDATION: ", formValidation)
-    }
+  function onLastNameChange(e) {
+    console.log(person);
+    setPerson({ ...person, lastName: e.target.value, lastNameTouched: true });
+    console.log('VALIDATION: ', formValidation);
+  }
 
-    function onLocationChange(e) {
-        console.log(person);
-        setPerson({ ...person, location: e.target.value })
-    }
+  function onLocationChange(e) {
+    console.log(person);
+    setPerson({ ...person, location: e.target.value });
+  }
 
-    function onRoleChange(e) {
-        setPerson({ ...person, role: e.target.value });
-        console.log(person);
-    }
+  function onRoleChange(e) {
+    setPerson({ ...person, role: e.target.value });
+    console.log(person);
+  }
 
-    function onHeadLineChange(e) {
-        setPerson({ ...person, role: e.target.value });
-        console.log(person);
-    }
+  function onHeadLineChange(e) {
+    setPerson({ ...person, role: e.target.value });
+    console.log(person);
+  }
 
-    function onTermsAndConditionChange(e) {
-        console.log(person);
-        setPerson({ ...person, termsAndConditions: !person.termsAndConditions });
-    }
-    function onJoinNewsLetterChange() {
-        console.log("Join News")
-    }
+  function onTermsAndConditionChange(e) {
+    console.log(person);
+    setPerson({ ...person, termsAndConditions: !person.termsAndConditions });
+  }
+  function onJoinNewsLetterChange() {
+    console.log('Join News');
+  }
 
-    const onContinueClick = (e) => {
-        router.push({
-            pathname: '/users/me',
-            query: { onboarding: true }
-        }, '/users/me');
-        e.preventDefault()
-    }
+  const onContinueClick = (e) => {
+    const profileId = userData?.profile?.id;
 
-    const onCancelClick = (e) => {
-        router.push({
-            pathname: '/',
-        }, '/');
-        e.preventDefault()
-    }
+    const requestBody = {
+      data: {
+        displayName: `${person.firstName} ${person.lastName}`,
```

Comment:
Would user be able to change this later?

Reply:
I am not sure if there are other ways to change the first and last name other than through the onboarding landing page but yes.